gnu.cpp

```diff
@@ -30,8 +30,7 @@ class Cowboy
   int Age;
 };
 
-Cowboy::Cowboy ()
-    : Age (45), GunCount (2)
+Cowboy::Cowboy () : Age (45), GunCount (2)
 {
```

Comment:
mmm, i tried to find the change in the coding standard, but could not find it. can you help me out an provide a reference?